tests/vmi_configuration_test.go

```diff
@@ -347,6 +347,49 @@ var _ = Describe("Configurations", func() {
 				Expect(err).ToNot(HaveOccurred())
 
 			})
+		})
+
+		Context("[rfe_id:140][crit:medium][vendor:cnv-qe@redhat.com][level:component]with support memory over commitment", func() {
+			It("[test_id:755]should show the requested memory different than guest memory", func() {
+				vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(tests.ContainerDiskFor(tests.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n")
+				guestMemory := resource.MustParse("256Mi")
+				vmi.Spec.Domain.Resources.Requests[kubev1.ResourceMemory] = resource.MustParse("64Mi")
+				vmi.Spec.Domain.Resources.OvercommitGuestOverhead = true
+				vmi.Spec.Domain.Memory = &v1.Memory{
+					Guest: &guestMemory,
+				}
+
+				vmi, err := virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+				Expect(err).ToNot(HaveOccurred())
+				tests.WaitForSuccessfulVMIStart(vmi)
+
+				expecter, err := tests.LoggedInCirrosExpecter(vmi)
+				Expect(err).ToNot(HaveOccurred())
+				defer expecter.Close()
+
+				res, err := expecter.ExpectBatch([]expect.Batcher{
+					&expect.BSnd{S: "free -m | grep Mem: | tr -s ' ' | cut -d' ' -f2\n"},
+					&expect.BExp{R: "236"},
```

Comment:
Testing that exactly 236M is free makes me a bit nervous. Would you be open to something along the lines of:

```
[ "$(free -m | grep Mem: | tr -s ' ' | cut -d' ' -f2)" -gt 200 ] && echo "pass" || echo "fail\n"
```

Where we test for something more general like "more than 200M free"?

Reply:
This minimal expected free memory of the guest directly depends on one of the test inputs. Either calculate it as, e.g., 78% of total guest memory, or specify it in a constant block together with the other test inputs.

```go
const (
    requested_mem = "64Mi"
    guest_mem = "256Mi"
    min_expected_free_guest_mem = 200
)
```

Reply:
@stu-gott @gszasz good points, thanks.

Reply:
done, but not with constants approach. I used `By` and description field from `Expect`